Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -359,6 +359,7 @@
                                                'keep HBHEDataFramesSorted_hcalDigis_*_*',
                                                'keep HFDataFramesSorted_hcalDigis_*_*',
                                                'keep HODataFramesSorted_hcalDigis_*_*',
+                                               'keep QIE10DataFrameHcalDataFrameContainer_hcalDigis_*_*',
```

Comment:
Seeing how this shows up in the list of products, I think it would be much better to have "HF" in the class name or, alternatively, in the product instance.
Is there a chance we end up with QIE10 in more than just HF?